cloud/jenkins/psmdb_operator_eks.groovy

```diff
@@ -254,7 +254,9 @@ EOF
             steps {
                 runTest('storage')
                 runTest('self-healing')
+                runTest('self-healing-chaos')
                 runTest('operator-self-healing')
+                runTest('operator-self-healing-chaos')
```

Comment:
no self-healing-advanced-chaos ?

Reply:
it seems that for PSMDB that test doesn't exist